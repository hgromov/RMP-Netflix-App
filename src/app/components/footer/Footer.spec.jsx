/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './Footer';

describe('Footer', () => {
  it('should match snapshot', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render component', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper.find('.Footer').length).toBe(1);
  });
});
