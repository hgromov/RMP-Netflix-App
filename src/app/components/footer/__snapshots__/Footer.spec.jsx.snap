// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match snapshot 1`] = `
<div
  className="Footer"
>
  <div
    className="Footer__logo logo"
  />
</div>
`;
